package.json

```diff
@@ -9,13 +9,17 @@
   },
   "scripts": {
     "start": "node src/app.js",
+    "start:ci": "API_BASE_URL=http://127.0.0.1:8090 REDIS_SESSION_URL= NODE_ENV=development node src/app.js",
     "start:debug": "DEBUG=* node src/app.js",
     "build": "yarn build-sass && yarn build-js",
     "build-sass": "rm -rf dist/public/style.css && sass --no-source-map src/assets/scss/application.scss dist/public/stylesheets/application.css --style compressed",
     "build-js": "yarn build-js:application && yarn build-js:all",
     "build-js:application": "mkdir -p dist/public/javascripts; uglifyjs src/assets/javascripts/application.js --beautify -o dist/public/javascripts/application.js",
     "build-js:all": "mkdir -p dist/public/javascripts; uglifyjs node_modules/govuk-frontend/govuk/all.js node_modules/hmpo-components/all.js --beautify -o dist/public/javascripts/all.js",
-    "test": "node_modules/.bin/jest --testPathPattern=app/bav --coverage"
+    "mocks": "yarn run wiremock --verbose --port 8090 --root-dir test/mocks",
+    "test:browser:report": "node ./test/utils/multiReportGenerator.js",
+    "test:browser": "wait-on tcp:127.0.0.1:8090 tcp:127.0.0.1:5040 && API_BASE_URL=http://127.0.0.1:8090 cucumber-js test/browser/features/**/*.feature --require test/browser/support/env.js  --require test/browser/support/setup.js --require test/browser/step_definitions/**/*.js -f json:./test/reports/cucumber_report.json",
```

Comment:
Think we are missing "wait-on" dependency

Reply:
"wait-on" dependency added